product-service/serverless.yml

```diff
@@ -1,13 +1,19 @@
 service: product-service
 frameworkVersion: '2'
+variablesResolutionMode: 20210219
 
 provider:
   name: aws
   runtime: nodejs12.x
-  stage: '${opt:stage}'
+  stage: ${opt:stage, 'dev'}
   region: us-east-1
   environment:
-    ENV_STAGE: '${opt:stage}'
+    ENV_STAGE: ${opt:stage, 'dev'}
```

Comment:
let's add todo comment here that this variables will be moved out to KMS or similar services